### ISSUE TITLE
livewire.js returns 404 in network logs

### ISSUE THREAD

**Issue by rebbieboi** [laravel/pulse#131]
### Pulse Version

^1.0@beta

### Laravel Version

10.34.2

### PHP Version

8.1.26

### Database Driver & Version

mysql  Ver 14.14 Distrib 5.7.35-38, for debian-linux-gnu (x86_64) using  7.0

### Description

when going to /pulse the page is loaded but no data is displayed and the elements aren't properly created.
Network logs show: 404 not found to the GET request to /livewire/livewire.js?id=123

this is a fresh install for pulse on an existing system. I have not used livewire before and wasn't aware it is in the system as well.

![image](https://github.com/laravel/pulse/assets/714513/3cea1380-d1a1-4fe5-b25c-04b1adbe5e63)
![pulse](https://github.com/laravel/pulse/assets/714513/0ce90442-220d-4f69-a1e0-40ef0a637c58)

### Steps To Reproduce

*

**Comment by rebbieboi** [comment]
managed to make it work following the comments in the linked issue. but my problem now is that nothing is recorded in the DB even after I execute "php artisan pulse:check". The DB is hosted on a different server since it is a cluster.

**Comment by stevethomas** [comment]
I ran into this right away; no data is rendering on the dashboard but everything else seemingly fine (db entries, no console errors, no exceptions etc). 

I made a dodgy fix by publishing the dashboard and adding `@livewireScripts`. 

This shouldn't be necessary because #88 (although I don't have this in vendor so maybe the PR was merged after the beta2 release?)  and https://livewire.laravel.com/docs/installation#manually-including-livewires-frontend-assets should be auto-injecting anyway. 

The only thing I can add is that I only just upgraded to LW3, and the LW scripts are not getting injected in my regular app either (`livewire.inject_assets` is `true`). 

I need to dig more, but i'm pretty sure it has something to do with https://github.com/livewire/livewire/blob/main/src/Features/SupportAutoInjectedAssets/SupportAutoInjectedAssets.php#L24   and the `$handled` variable having some very strange behaviour (`ray($handled)` outputs nothing), so this is not looking like a Pulse-specific issue to me.
